### PR TITLE
November 2023 Meeting Minutes PR

### PR DESCRIPTION
**Pull request by surfersol**
Please note that PlebNetDev_Meeting_Notes_Oct2023.md has a description of "November 2023 Meeting Minutes" and should be "October 2023 Meeting Minutes"

### CODE REVIEW

**Comment by bliotti** [comment]
@surfersol do you need me to resolve the conflicts?

**Comment by bliotti** [comment]
Assuming I am cool to merge? Does the website just reference this repo's md files now?

**Comment by surfersol** [comment]
> Question - Assuming I am cool to merge?   Answer - yes if all well.

Question -Does the website just reference this repo's md files now? 
Answer - repo's md is not in the website of November blog as this PR needs to be approved and then I can put this November meeting minute's website to the the November blog.

Question - I don't see any pipeline stuff for this repo.
Answer - Unsure in how to answer or create a pipeline stuff for this repo.  Do you have to approve and merge to the main website?

File: PlebNetDev_Meeting_Notes_Nov2023.md [hunk 1]
```diff
@@ -0,0 +1,76 @@
...
+#### 1.	**Projects Updates**
+
+#### 2.	Plebnet.dev Updates 
+- [Member Contributions for October 2023](https://plebnet.dev/october_2023_collaborators)
+
+- [Member Contributions for November 2023](https://plebnet.dev/november_2023_collaborators)
+
+#### 3.	Project Ideas  
+
+[Nostr mail](https://github.com/asherp/nostr-mail/tree/app) in association with [kivyMD](https://github.com/kivymd/KivyMD/blob/master/README.md)
```
Comment (bliotti) [inline PlebNetDev_Meeting_Notes_Nov2023.md:31]: make all these bullet points

File: PlebNetDev_Meeting_Notes_Nov2023.md [hunk 2]
```diff
@@ -0,0 +1,76 @@
...
+[Aionostr](https://github.com/davestgermain/aionostr)
+
+[Noobs guide to adding #zaps on #nostr](https://snort.social/e/note1y475m5qw5f7xpsh8azrxcm5ywyptqwyphnkzdzlzxg0akqld02dqs2w46u)
+
+[bitcoinJ is a library for working with the Bitcoin protocol. It can maintain a wallet, send/receive transactions without needing a local copy of Bitcoin Core and has many other advanced features. It's implemented in Java but can be used from any JVM compatible language: examples in Python and JavaScript are included](https://bitcoinj.org/)
+
+[Nix Project](https://github.com/NixOS/nix)
+
+#### 4.	Learning and Development
+    Prospect Workshops
```
Comment (bliotti) [inline PlebNetDev_Meeting_Notes_Nov2023.md:42]: remove indention

File: PlebNetDev_Meeting_Notes_Nov2023.md [hunk 3]
```diff
@@ -0,0 +1,76 @@
...
+
+- [Bitcoin J](https://bitcoinj.org/) & Java by Msgilligan
+
+#### 5.	Community & Collaborative Initiatives 
+
+#### 6.	Additional Topics & Technological Interests:
+
+- Alby community to work on projects.  Go to technical-sessions to make an appointments
+
+#### 7.	Other Discussion Points
```
Comment (bliotti) [inline PlebNetDev_Meeting_Notes_Nov2023.md:58]: make bullet points

File: PlebNetDev_Meeting_Notes_Nov2023.md [hunk 4]
```diff
@@ -0,0 +1,76 @@
...
+[Completion of a LNBITS or GetAlby Pull Request would get a free 2024 Bitcoin Tix (Open Source](https://b.tc/conference/opensource)
+
+[A Hybrid Solution To Unjamming Lightning - Main Stage TABconf 2023](https://www.youtube.com/watch?v=ATPtbO3awcQ)
+
+#### 8.  Action Items:  
+
+#### 9/  Previous Meeting Minutes (https://github.com/plebnet-dev/meeting-notes)
```
Comment (bliotti) [inline PlebNetDev_Meeting_Notes_Nov2023.md:65]: fix slash

File: PlebNetDev_Meeting_Notes_Nov2023.md [hunk 5]
```diff
@@ -0,0 +1,76 @@
...
+[Noobs guide to adding #zaps on #nostr](https://snort.social/e/note1y475m5qw5f7xpsh8azrxcm5ywyptqwyphnkzdzlzxg0akqld02dqs2w46u)
+
+[bitcoinJ is a library for working with the Bitcoin protocol. It can maintain a wallet, send/receive transactions without needing a local copy of Bitcoin Core and has many other advanced features. It's implemented in Java but can be used from any JVM compatible language: examples in Python and JavaScript are included](https://bitcoinj.org/)
+
+[Nix Project](https://github.com/NixOS/nix)
+
+#### 4.	Learning and Development
+    Prospect Workshops
+
+- Designing your apps with [KivyMD]((https://github.com/kivymd/KivyMD/blob/master/README.md) by (@Asherp)[https://github.com/asherp]
```
Comment (bliotti) [inline PlebNetDev_Meeting_Notes_Nov2023.md:44]: fix

**Review by bliotti — CHANGES_REQUESTED** [review]

**Pull request merged**